testing/unittests/esdltests.cpp

```diff
@@ -0,0 +1,666 @@
+/*##############################################################################
```

Comment:
No tests for alternate, legacy, operation names. No tests for bad input (see earlier comments for examples).